.github/workflows/soroban-rpc.yml

```diff
@@ -61,7 +61,9 @@ jobs:
         with:
           go-version: 1.20.1
 
-      - run: rustup target add ${{ matrix.rust_target }}
+      - run: |
+          rustup target add ${{ matrix.rust_target }}
+          rustup update
```

Comment:
do you know where/when rustup is installed initially, only ask b/c depending on how it's installed, the self `update` may not be allowed - https://rust-lang.github.io/rustup/basics.html#keeping-rustup-up-to-date, but doubt that will ever be case here, this seems fine. the other test container setups I've seen so far, they just download rustup with `--default-toolchain` with equates to the latest `stable` which is same as `update`

Reply:
I had this same question - I initially assumed I should be able to tweak the installed version in some `install-rust`esque action, but no such luck?? I guess it's part of `ubuntu-latest`, which is surprising. I assume this will self-resolve if the image bumps its installed version, too? But since this is a "good enough" solution I'm just gonna merge.